apidoc/Titanium/UI/TextArea.yml

```diff
@@ -88,6 +88,8 @@ properties:
         The underlying attributed string drawn by the textArea. If set, avoid setting common attributes
         in textArea, such as `value`, `color` and `font`, as unexpected behaviors may result.
 
+        This has no effect when used with `hintType` Ti.UI.HINT_TYPE_ANIMATED.
+
```

Comment:
Remove this sentence from the "attributedString" property documentation. Hint type does not apply to it.

The TextArea documentation is missing the "attributedHintText" property.